app/controllers/message_threads/bulk/exports_controller.rb

```diff
@@ -17,7 +17,12 @@ def edit
         authorize @export
 
         @message_threads = message_thread_policy_scope.where(id: @export.message_thread_ids).includes(:messages)
-        @message_types = ["default"] + @message_threads.flat_map do |thread|
+        @message_forms = @message_threads.map do |thread|
+          thread.metadata['fs_form_id']
+        end.then do |ids|
+          Fs::Form.where(id: ids).pluck(:slug)
```

Comment:
Tu som objavila jednu vec. Ak exportujes napr. DPH riadny a DPH opravny, slug je rovnaky, ale u nas su to dva rozne Fs::Formy, takze v GUI to vidno duplicitne (u mna ukazane na inom formulari): 
<img width="1354" height="265" alt="image" src="https://github.com/user-attachments/assets/bc83a3a1-a60a-4cd5-a257-398c32220a6e" />

@jsuchal